Library/Homebrew/system_command.rb

```diff
@@ -427,6 +427,7 @@ def success?
     def to_ary
       [stdout, stderr, status]
     end
+    alias to_a to_ary
```

Comment:
Probably better to use `#stdout`, `#stderr` or `#status` instead of explicitly calling `#to_a`.

Reply:
The only downside is that using the explicit methods could potentially require up to four lines if using all of the values in a way that requires them to be stored in variables. However, it doesn't seem like that will be a common issue in practice, so I'm good with this approach.

For example, the `curl_output` call in `Utils::GitHub::API.open_rest` uses `output`, `errors`, and `status`, so we would need to store the return value in an intermediate variable (e.g., `result`) to be able to use all of the values. If we replace `errors` with `result.stderr` and `status.success?` with `result.status.success?`, we end up using the same number of lines.

I'll address this in the next push as well 👍